src/LoginPage/LoginForm.test.jsx

```diff
@@ -31,6 +33,20 @@ describe('LoginForm', () => {
 
       expect(handleClick).toBeCalled();
     });
+
+    it('listens change events', () => {
+      const { getByPlaceholderText } = renderLoginForm({
+        isLogin: true,
+      });
+
+      const input = getByPlaceholderText('Email');
+      const value = { value: 'test@test.com' };
```

Comment:
```suggestion
const value = 'test@test.com';
```
그냥 이렇게 작성하셔도 똑같이 동작할 거 같아요!!🙂

Reply:
아앗 그렇네요!
테스트 코드의
```jsx
expect(handleChange).toBeCalledWith({ name, value });
```
에 맞게 해주려다가 RORO 패턴과 혼동햇네요! 🥹